service/src/main/resources/application.yml

```diff
@@ -48,4 +48,10 @@ gp2gp:
     rootCA: ${GP2GP_SPINE_ROOT_CA_CERT:}
     subCA: ${GP2GP_SPINE_SUB_CA_CERT:}
   mhs:
-    url: ${GP2GP_MHS_OUTBOUND_URL:http://localhost:8081/mock-mhs-endpoint}
\ No newline at end of file
+    url: ${GP2GP_MHS_OUTBOUND_URL:http://localhost:8081/mock-mhs-endpoint}
+  sds:
```

Comment:
Can we update the README.md to describe each of these?

Also, what is the purpose of the proxy settings. Does SDS API require a proxy?

Reply:
Readme updated. Proxy removed